docs/HowtoCreateCollectionsOrRelatedLinksInMetalsmith.md

```diff
@@ -0,0 +1,104 @@
+# Adding a collection of related links
+_Added: 2017-04-27_
+
+Our current publishing system, Metalsmith, provides a way to group related content using what it calls _collections_. Using collections requires the `metalsmith-collections` plugin ([documentation](https://github.com/segmentio/metalsmith-collections)).
+
+Content grouped within a collection is displayed in the sidebar navigation. Viewing `content/components/navigation-sidebar.html` may clarify some of what's in this document.
```

Comment:
Should the fact that they're also used to define the previous/next navigation be mentioned here too?